src/Uno.UI/UI/Xaml/UIElement.Pointers.wasm.cs

```diff
@@ -143,9 +143,10 @@ public partial class UIElement : DependencyObject
 #if NET7_0_OR_GREATER
 	[JSExport]
 #endif
+#pragma warning disable IDE0055 // Fix formatting
```

Comment:
The reason you are seeing this is because the BlockSyntax inside the "case" is not indented. This behavior can be controlled via `csharp_indent_case_contents_when_block` in `.editorconfig` (the property defaults to true). The following is example from the docs that shows how this option controls formatting.

```
// csharp_indent_case_contents_when_block = true
case 0:
    {
        Console.WriteLine("Hello");
        break;
    }

// csharp_indent_case_contents_when_block = false
case 0:
{
    Console.WriteLine("Hello");
    break;
}
```

If the desired formatting is to not indent the block, then we should adjust the property in .editorconfig rather than disabling the analyzer with `#pragma`.

Reply:
Set this to `csharp_indent_case_contents_when_block = false:suggestion` so it won't force one over the other.

Reply:
That *will* still force the "false".

If this produced too many violations in CI, we may keep the consistency for now and indent the block, then adjust for the whole codebase in a follow-up PR.

Reply:
It works on my computer ... let see on the CI :)